std-bits/table/src/main/java/org/enso/table/data/column/operation/map/numeric/DoubleRoundOp.java

```diff
@@ -0,0 +1,84 @@
+package org.enso.table.data.column.operation.map.numeric;
+
+import org.enso.polyglot.common_utils.Core_Math_Utils;
+import org.enso.table.data.column.operation.map.TernaryMapOperation;
+import org.enso.table.data.column.operation.map.MapOperationProblemBuilder;
+import org.enso.table.data.column.storage.numeric.DoubleStorage;
+import org.enso.table.data.column.storage.numeric.LongStorage;
+import org.enso.table.data.column.storage.Storage;
+import org.enso.table.error.UnexpectedTypeException;
+import org.graalvm.polyglot.Context;
+
+import java.util.BitSet;
+
+/** An operation rounding floating-point numbers. */
+public class DoubleRoundOp extends TernaryMapOperation<Double, DoubleStorage> {
+
+    public DoubleRoundOp(String name) {
+        super(name);
+    }
+
+    @Override
+    public Storage<?> runTernaryMap(DoubleStorage storage, Object decimalPlacesObject, Object useBankersObject, MapOperationProblemBuilder problemBuilder) {
+        if (decimalPlacesObject == null || useBankersObject == null) {
+            return DoubleStorage.makeEmpty(storage.size());
+        }
```

Comment:
Why this behaviour? Are these parameters _ever_ allowed to be null?? I thought not.

I would throw an exception that this is rather unexpected.

Reply:
Throwing IllegalArgumentException now.